examples/onebrc/README.md

```diff
@@ -0,0 +1,216 @@
+
+./proton client --host 127.0.0.1 --multiquery < 1brc.sql  0.28s user 0.05s system 0% cpu 39.301 total
+</pre>
+</details>
+
+The Proton server reported a summary of the executed query in its console output:
+```bash
+2024.04.13 23:07:16.705492 [ 14810 ] {33215e8b-670e-464b-a23e-994f706b7a64} <Information> executeQuery: Read 1000000000 rows, 19.51 GiB in 39.038123 sec., 25615985 rows/sec., 511.89 MiB/sec.
```

Comment:
Just FYI, on my Mac, it was 

> executeQuery: Read 1000000000 rows, 19.51 GiB in 21.283514 sec., 46984722 rows/sec., 938.91 MiB/sec

Apple M2 Pro, 10 cores (6 performance and 4 efficiency), 32GB memory.

But we don't need to take the numbers to seriously, there are other apps running, too many factors. But on the same machine, similar workload, comparing Timeplus Proton vs ClickHouse or other sql engines are okay.

Reply:
We could use your run times for the final article since it's faster 😀.

My run was on a 2020 Intel MacBook Pro with 16GB of RAM.

Reply:
I am really surprised we do better in the csv aspect.
if we really care the perf, we shall tag the proton version and clickhouse version, also the hardware spec as well.
otherwise, it smells a bit unfair/unclear to reproduce.

Reply:
@yokofly you make a fair point.

The easiest way out here would be to add the specific software versions used and my hardware specs to the article.

I'd added my machine's specs and the versions used as footnotes so anyone who wants to reproduce it will be able to do so. You can see the changes in this commit: [653e4bb](https://github.com/ayewo/proton/commit/653e4bbb979366cf7393bbeaaab41f9ce8c85147).